docs/UserGuide.md

```diff
@@ -234,28 +263,112 @@ Oops! There seems to be an error, please check your fields again.
 
 Deletes the data entry at the index-th position of the currently displayed list.
 
-Format: `delete INDEX...`
+Format: `delete INDEX [INDEX...]`
+
+Parameter: `INDEX [INDEX...]`
+* Index of a fosterer is displayed in the list that is  obtained by the `find/list` command.
+* At least one index must be provided.
+* Index must be a positive integer: 1, 2, 3, …
+
+<div markdown="span" class="alert alert-primary">
+  :bulb: <b>Tip:</b>
+Multiple indices are allowed for mass deletion, each index separated by a white space.
+</div>
+
+* Duplicate indices are ignored.
+* Extraneous white spaces are also ignored.
 
-Parameter: `INDEX...`
-* Index of a fosterer is shown in the list obtained by the `find/list` command.
-* Must be a positive integer 1, 2, 3, …
-* Multiple indexes are allowed for mass deletion,  each index separated by a white space.
-* `delete` without an index is valid if and only if there exists only one data entry in the current list.
 
 Examples:
 * `list` followed by `delete 2` deletes the 2nd fosterer in the address book
-* `find Jerry` followed by `delete 1` deletes the 1st fosterer in the results of the find command
+* `find Jerry` followed by `delete 1` deletes the 1st fosterer in the result list of the find command
```

Comment:
i think can specify "of the find/list command"

Reply:
ok done!